### DISCUSSION TITLE
Alphabet-Indexed Scroll Bar for Quick Navigation to Items by Alphabetically

### DISCUSSION THREAD

**Discussion by BlwAvg** [advplyr/audiobookshelf#2532] [Ideas]
**Proposed Solution:**
The proposed alphabet-indexed scroll bar will enhance the user experience by:

- Allowing for rapid navigation to items starting with a specific letter.
- Displaying a clear, visual indicator of the current section of the list (alphabetically).
- Reducing the time and effort needed to locate specific items in long lists.

**Implementation Details:**
- The scroll bar will be visible alongside the standard scrolling mechanism.
- As the user scrolls through the list, a highlighted letter will appear next to the scroll bar, changing dynamically to reflect the items currently in view.
- Users can also click on a specific letter along the scroll bar to jump directly to that section of the list.

**Comment by nichwall** [comment]
Yeah, it's been brought up a few times. I think there's another discussion/issue somewhere, but can't find it at the moment.

https://github.com/advplyr/audiobookshelf/issues/2165
https://github.com/advplyr/audiobookshelf-app/issues/544

**Reply by BlwAvg** [reply]
Well, look at that. I looked but I guess I did not look hard enough. Thank you Nichwall.
